pkg/storaged/crypto-keyslots.jsx

```diff
@@ -267,6 +267,14 @@ export function init_existing_passphrase(block, just_type, callback) {
    do this unconditionally because it's so fast.
 */
 
+function check_installed_package(name) {
+    // This needs to work without PackageKit, so that people can make
+    // progress by manually installing a package and trying again in Cockpit.
+    return cockpit.spawn(["rpmquery", name], { err: "message" })
+            .then(() => true)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18698-20230425-113621-0ab0dcb3-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)